crates/subspace-consensus-runtime/src/lib.rs

```diff
@@ -71,12 +71,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("subspace"),
     impl_name: create_runtime_str!("subspace"),
     authoring_version: 1,
-    // The version of the runtime specification. A full node will not attempt to use its native
-    //   runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`,
-    //   `spec_version`, and `authoring_version` are the same between Wasm and native.
-    // This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
-    //   the compatible custom types.
-    spec_version: 100,
```

Comment:
Did you check that this doesn't break Polkadot.js apps? That is the only reason I left it as is.

Reply:
I have run the local node with Polkadot.js apps, seems to be fine. Any more concrete steps to ensure this does not break it?

Reply:
If all of the types are decoded properly I think it is a sufficient test.